Submissions/Hamidmirzapour.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "hamid Mirzapour",
```

Comment:
Should `hamid` be capitalized?

Reply:
Sure, Thanks.

Reply:
In my last commit, I did what you want. Please pull my request. 
Thanks a lot.